### ISSUE TITLE
Pull Request and Commits information showing incorrect Author/Reviewer avatars

### ISSUE THREAD

**Issue by Agus** [atlassian/github-for-jira#1828]
For some reason the avatars of the users that are Authors or Reviewers in the Pull Request and Commits sections of an Issue Detail View are incorrect, I don't know where they are being taken from 😐, for example for my user is using an avatar that I don't have in either Jira (or any Atlassian product) or GitHub, but when I inspect from where it's coming from `avatar-management--avatars.us-west-2.prod.public.atl-paas.net`, so looks like it's stored _somewhere_ in the Atlassian infrastracture.

I would like to know from where you're taking this avatars and how we can fix this thing of showing avatars that were **not provided** to either software. Thanks!

**Comment by Agus** [comment]
Ooookay, figured out what was going on after creating the issue.. If someone's having the same issue basically it's because you have the same setup as I have which is:
- You're using Atlassian / Jira with your **work email**
- You push code with your **personal account** with GitHub
The trick is that it seems that I have some **personal account** in Jira as well (I don't remember creating it tbh 😅). So when it loads the PRs / Commits it checks the author, and since my personal email matches in both GitHub and this "personal Jira account", it was showing some profile avatar that I had there.

Long story short, it was my fail because I didn't know I had a personal account on Jira. 😂
